setup.py

```diff
@@ -1,6 +1,6 @@
 from setuptools import setup, find_packages
 import pathlib
-from gpmf import __version__
+__version__ = "0.1"
```

Comment:
Great! Remove the `__version__` definition in `gpmf/__init__.py` too so we don't have duplicate definitions that risk being out of sync.